src/main/scala/com/wavesplatform/state/appender/package.scala

```diff
@@ -97,9 +97,9 @@ package object appender extends ScorexLogging {
     val fs        = bcs.functionalitySettings
     val blockTime = block.timestamp
     val generator = block.signerData.generator
+    val height    = blockchain.height
```

Comment:
Hm.. We rely on BlockchainUpdaterImpl here, it checks the referenced block, right?

Reply:
This piece was broken, actually. The `height` used to be the height of the `block`'s parent, but it was not necessarily the height of the last block.